manifests/server.pp

```diff
@@ -59,6 +59,9 @@
   $smtpd_sasl_path = 'private/auth',
   $smtp_sasl_auth = false,
   $smtp_sasl_password_maps = undef,
+  # sende-dependent relay
+  $smtp_sender_dependent_authentication = false,
```

Comment:
Line 62 and 63 are not required. You can check, if `sender_dependent_relayhost_maps` is set within the template, as line 63 isn't changing anything else

Reply:
Line 62 and 63 have been removed.